skbio/stats/composition.py

```diff
@@ -107,6 +107,8 @@
 import scipy.stats
 import skbio.util
 from skbio.util._decorator import experimental
+from collections import OrderedDict
```

Comment:
Why use `OrderedDict`? I think that dictionaries are ordered by default since Python 3.7. https://stackoverflow.com/questions/39980323/

Reply:
This is pre-Python 3.7 legacy code -- but I did not know about the ordering by default, very cool!